Composer/packages/client/src/pages/botProject/adapters/ExternalAdapterModal.tsx

```diff
@@ -35,18 +35,24 @@ type Props = {
   projectId: string;
   schema: JSONSchema7;
   uiSchema: JSONSchema7 & { helpLink?: string };
-  value?: { [key: string]: ConfigValue };
+  value?: { [key: string]: JSONSchema4Type };
```

Comment:
Why use `JSONSchema4Type` instead of the already imported `JSONSchema7` type?

Also, this could be replaced by `SchemaDefinitions` which is also exported by the types package.

Reply:
Good idea - this was to fix a typecheck error, so I pulled the first thing I found that looked like what it was complaining about.

Reply:
Ok. Not a big deal if that doesn't correct the issue.